tracing-core/src/field.rs

```diff
@@ -224,6 +224,11 @@ pub trait Visit {
         self.record_debug(field, &value)
     }
 
+    /// Visit a byte slice.
+    fn record_byte_slice(&mut self, field: &Field, value: &[u8]) {
+        self.record_debug(field, &value)
```

Comment:
it might be nice to have the default `record_debug` call format the bytes in hexadecimal rather than in decimal, since code that records a byte slice is probably trying to record binary data rather than a list of integers?

Reply:
I changed it to writing out the hex representation in packed way, e.g. `a7e93f`. You didn't mean just changing decimal to hexadecimal as in `[a7, e9, 3f]`, right?